src/quick-lint-js/error.h

```diff
@@ -260,12 +260,38 @@
       error_unexpected_control_character, { source_code_span character; },     \
       .error(QLJS_TRANSLATE("unexpected control character"), character))       \
                                                                                \
```

Comment:
Line 254-257: Could we delete `error_unexpected_characters_in_number`? Is it used anywhere still?

Reply:
it's still used at [lex::parse_numbrer](https://github.com/catac1/quick-lint-js/blob/5c835c280253897dd4095007ba19f639e1cecac1/src/lex.cpp#L846) the test case for it are [test-lex.cpp#L320](https://github.com/catac1/quick-lint-js/blob/5c835c280253897dd4095007ba19f639e1cecac1/test/test-lex.cpp#L320).